packages/cli/src/__tests__/project/web3api.buildx.yaml

```diff
@@ -0,0 +1,16 @@
+format: 0.0.1-prealpha.3
+docker:
+  dockerfile: ./Dockerfile
+  buildkit:
+    enabled: true
```

Comment:
Isn't this redundant? If `buildkit:` is present, you're already saying "I want buildkit enabled", correct?

Reply:
What if the user wants to just enable buildkit without any custom options? (Ex: cache, output)

Reply:
Nvm We can do similar to what you suggested below, `buildkit: Boolean | BuildkitOptions`